data/human/news.txt

```diff
@@ -1198,6 +1270,42 @@ news "human farming in pirate space"
 
 
 
+news "syndicate pa system"
+	location
+		government "Syndicate"
+		not attributes station
+	name
+		word
+			"Public Announcement System"
+	message
+		word
+			"Attention: baggage should not be left unattended. Any unattended baggage will be auctioned within days."
+			"Attention factory workers: please proceed to the fast-lane immigration desk. Time spent in regular immigration lanes will not be reimbursed."
+			"Dear spaceship captains: please be aware that our outfitters are open all hours of the day. There is never a reason to delay the purchase of your next piece of Syndicate equipment."
+			"Dear traveler: we recommend you to have dinner in our local restaurant 'The Wing'. You will not be disappointed."
```

Comment:
These are still spoken messages, so this phrase should have quotation marks at the top and bottom.